src/styles/main.scss

```diff
@@ -1,3 +1,10 @@
-body {
-  margin: 0;
-}
+@import "./blocks/variables/variables";
+
+@import "./blocks/head-block/icon";
+@import "./blocks/head-block/header";
+@import "./blocks/head-block/nav";
+
+@import "./blocks/card-block/content-card";
```

Comment:
Styles for the 'card-block' seem to be split into two files: 'content-card' and 'stars'. If these are related, it's better to keep them in one file to maintain Readability and Cohesion. Only split if they are used separately in different contexts.